test/fw/ptl/utils/pbs_logutils.py

```diff
@@ -274,30 +271,37 @@ def convert_date_time(cls, dt=None, fmt=None, syslog=False):
         else:
             return int(tm)
 
-    def get_syslog_date_str(self, logline=None):
-        if logline is None:
-            return None
-
-        if "T" in logline:
-            return logline[:19]
-        else:
-            return logline[:15]
-
-    def convert_syslog_time(self, dt=None):
-        if dt is None:
-            return None
+    def convert_syslog_date_str(self, logline):
+        """
+        Convert syslog time string in logline to the  str time format
+        required by the convert_date_time(),
+        which will convert it to epoch time.
+        The syslog time can be in 2 formats-
+        1) RFC 5424- "yyyy-mm-ddThh:mm:ss-TZ"
+        2) "Month date hh:mm:ss" (with no year)
```

Comment:
are 'month' and 'date' in a particular format? e.g - is 'month' 3 letters? (jan, feb, etc.) or is it 2 letter numeric? (01, 02, etc.) ? what about date? Please mention the exact format here